news/2022-08-01-osumonthly-1.md

```diff
@@ -0,0 +1,208 @@
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/byyHOAopo1A" frameborder="0" allowfullscreen></iframe>
+
+---
+
+## New Features on osu!(lazer)!
+
+The new client has never so **close** from releasing (there's still a lot of work on it!), and has **new mods**, more original than never! 
+For the first time ever, lazer patch notes have been posted on the official osu! [YouTube channel](https://www.youtube.com/c/osugame), compiled by [pishifat](https://osu.ppy.sh/users/3178418) with further insight provided by the man himself, [peppy](https://osu.ppy.sh/users/2)!
+Check the video out:
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/67P4LKlAh-0" frameborder="0" allowfullscreen></iframe>
+
+If you are interested in all previous and future patch notes, you can find them [here](https://osu.ppy.sh/home/changelog).
+
+One thing you might've missed is the new fancy Repel mod added to lazer, which is pretty chaotic but fun (finally a worthy opponent for [badeu](https://osu.ppy.sh/users/1473890)?)
+
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://user-images.githubusercontent.com/47010459/172435116-6b09a48c-6fd6-4abb-9e5f-e96ab3565184.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+Another feature that's maybe not the newest, but some of you might have missed, or just haven't gotten around to trying yet is the **skin layout editor**! First implemented in May of 2021, this is still very much a work-in-progress feature, however currently you are able to edit the placement of all skin elements that appear during gameplay! Check out the showcase video from the original patch notes below:
+
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://user-images.githubusercontent.com/191335/118155608-5c5fce80-b453-11eb-88f4-e552b10ed5f1.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+## Month's Community Quiz
+
+A monthly question or puzzle for you!
+
+> Once upon a time, in the ye' old osu!, someone asked for advice from the ancestors among osu!, the post reads as follows: 
+
+> "Is it possible to repeat songs in osu!? I have an in-game video of KH2 with Roxas battling the Twilight Thorn (people who have played Kingdom Hearts 2 will be drooling) and the video is still halfway when the song ends! Or can someone at least tell me about a free, easy to use software which can solve my problem by repeating the song?"
+
+> This ancient tome is hidden behind a code, the code you seek is everywhere but no where at the same time, always in the back of your mind but only breaching the surface at the moment you see it, now... go forth, unveil the secret code, and paste the number in the link, and gain access to this ancient tome!
+
+> The first person to answer it correctly will receive a month of osu!supporter, leave your answer in the comment!
```

Comment:
```suggestion
The first person to answer it correctly will receive a month of osu!supporter, leave your answer in the comment!
```
I recommend to remove the block quotation for this one. It feels less as a part of the quiz itself and more as a piece for the user prompt.